tests/test_scheduler.py

```diff
@@ -391,16 +386,16 @@ def test_lsf2():
     expected = """#BSUB -P account_name
 #BSUB -q batch
 #BSUB -W 00:01:00
-#BSUB -R affinity[core(1)]
+#BSUB -n 12
```

Comment:
This test shouldn't be passing.
The number here should be 144, based on the `props` input.

Reply:
created issue